docs/UserGuide.md

```diff
@@ -4,9 +4,9 @@
   pageNav: 3
 ---
 
-# AB-3 User Guide
+# Clinic Mate User Guide
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a  Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+Clinic Mate is a **desktop app for managing contacts in a clinic, optimized for use via a  **Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, Clinic Mate can get manage your patients' contact faster than traditional GUI apps.
```

Comment:
Should it be "Clinic Mate can ~~get~~ manage your patients..." in the last sentence?

Reply:
yep, changed :), anymore?

Reply:
![image](https://github.com/AY2324S2-CS2103T-F14-2/tp/assets/110999129/5bde6e0c-10d8-43bc-8f19-5d2a37e14225)

I think there's an extra double asterisk behind desktop.